utilities.py

```diff
@@ -8,30 +14,41 @@
 
 
 def parameter_file_load(x):
-    try:
-        with open(x) as json_file:
-            text_input = pd.read_json(json_file)
-            return text_input
-    except IOError:
-        print("File Handling Error: ", IOError)
+    if os.path.exists(x):
+        try:
+            with open(x) as json_file:
+                text_input = pd.read_json(json_file)
+                return text_input
+        except Exception as e:
+            logging.error("There was issue to Open\Read the file Due to: " + str(e))

```

Comment:
what about raise the exception?    be aware that without raising it back it will be masked.

Reply:
1. unnecessary test was removed.
2. Exception change to IOError for handling the open\read Exception's
3. raise was added